src/plugins/kibana_utils/public/state_sync/state_sync_state_storage/create_kbn_url_state_storage.ts

```diff
@@ -63,7 +63,7 @@ export const createKbnUrlStateStorage = (
     onGetError,
     onSetError,
   }: {
-    useHash: boolean;
+    useHash?: boolean;
```

Comment:
This param is defaulted so I believe can be set as optional?

Reply:
Probably a viable change, but if code owner review becomes a blocker maybe we should consider reverting and landing the other small change.

Reply:
I'm not 100% sure why this was marked as required at the time, perhaps it is because this functionality was "experimental" and is often tied to an advanced setting, e.g.:

```ts
useHash: coreStart.uiSettings.get('state:storeInSessionStorage')
```

and the default would one day change.

@Dosant can confirm when is back next week. This looks like a safe change to make though, but I would advise waiting until he gets back next week or unblocking by not making this change now per @kpollich .

Reply:
Brill thanks @jloleysens I'll work around it for now 👍